html/manifest/shortcuts.json

```diff
@@ -0,0 +1,62 @@
+{
+  "html": {
+    "manifest": {
+      "shortcuts": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/Manifest/shortcuts",
+          "spec_url": "https://w3c.github.io/manifest/#shortcuts-member",
+          "support": {
+            "chrome": {
+              "version_added": "85",
+              "notes": "Supported on Windows only."
```

Comment:
The link to the Chrome status entry is broken, so I don't know the full story here, but I suspect this needs to be flagged as partial for https://github.com/mdn/browser-compat-data/blob/master/docs/data-guidelines.md#operating-system-limitations-imply-partial_implementation

Reply:
Good catch! Added :)